internal/config/config.go

```diff
@@ -58,6 +61,7 @@ func (config *DegitxConfig) FromFiles(paths ...string) error {
 	return nil
 }
 
+// FromFile parses yaml file.
```

Comment:
@stepanvalyavskiy some comments are inconsistent with other: some has dot at the end, other doesn't have. I'd remove dots from comments